Extensions/dnSpy.Debugger/dnSpy.Debugger.DotNet.Metadata/DmdLazyMetadataBytes.cs

```diff
@@ -151,4 +151,15 @@ public sealed class DmdLazyMetadataBytesCom : DmdLazyMetadataBytes {
 			Dispatcher = dispatcher ?? throw new ArgumentNullException(nameof(dispatcher));
 		}
 	}
+	/// <summary>
+	/// No metadata
+	/// </summary>
+	public sealed class DmdLazyMetadataBytesNull : DmdLazyMetadataBytes {
+		/// <summary>
+		/// Constructor
+		/// </summary>
+		public DmdLazyMetadataBytesNull() { }
+	}
+
+
 }
```

Comment:
```suggestion
}
```
Reason: These two blank lines are unnecessary.